frontend/src/helpers/connectionHelper.ts

```diff
@@ -49,6 +51,13 @@ export function clearConnectionError(connection: IConnection) {
   emit('connection', connection)
 }
 
+export function getConnectionIds(state: ApplicationState) {
+  const { device, connections } = state.backend
+  let ids = connections.map(c => c.id)
+  if (device.uid) ids.push(device.uid)
```

Comment:
could you add validation to check if the id is not duplicate? I am not sure

Reply:
Sure why not!